docs/network/fees.md

```diff
@@ -65,3 +67,11 @@ Etherlink calculates the inclusion fee with this equation:
 $$
 \texttt{0.000004 XTZ} * (150 + \texttt{tx.data.size()} + \texttt{tx.access\_list.size()})
 $$
+
+## Gas limit
+
+The gas limit, or the maximum total amount of execution fees in a single Etherlink block, is 30 million times the current gas price.
+Transactions that require a higher execution fee fail.
+
+There is no straightforward way of determining the execution fee for a transaction because estimating transaction fees with the `eth_estimateGas` endpoint the sum of the execution fee and the inclusion fee.
```

Comment:
```suggestion
There is no straightforward way of determining the execution fee for a transaction because estimating transaction fees with the `eth_estimateGas` endpoint the sum of the execution fee and the inclusion fee is returned as a single amount.
```